### PR TITLE
refactor(build): replace node-sass with dart sass

### PR DESCRIPTION
**Pull request by rapzo**
Proposal for replacing node-sass with dart sass.
`node-sass` was breaking fresh install, both on OSX and Linux with node v12.x.x (but works on node v10.x.x).
Dart implementation seems to be the official(?) one. And works on both node versions.

#### Changelog

**New**

* Dart sass, or `sass` node module.

**Changed**

* Build tools dependencies.

**Removed**

* `node-sass` as a devDependency

### CODE REVIEW

**Comment by rapzo** [comment]
Thank you and thanks for all the work! I'm loving using carbon!!!
And sorry for the random "update dependency" commit :laughing:

**Comment by cal-smith** [comment]
Thanks! and I'm glad you like it 💯

**Comment by zvonimirfras** [comment]
@rapzo Looks like there are conflicts in `package-lock.json`, do you mind resolving that so we can merge this? 😃

**Comment by rapzo** [comment]
Done!

File: package.json [hunk 1]
```diff
@@ -119,10 +118,11 @@
     "karma-coverage-istanbul-reporter": "2.0.5",
     "karma-jasmine": "2.0.1",
     "karma-jasmine-html-reporter": "1.4.2",
-    "node-sass": "4.11.0",
+    "ng-packagr": "5.3.0",
     "postcss-loader": "3.0.0",
     "raw-loader": "1.0.0",
     "rxjs": "6.4.0",
+    "sass": "^1.22.12",
```
Comment (zvonimirfras) [inline package.json]: Let's lock this version too, to avoid accidental breakages.
```suggestion
    "sass": "1.22.12",
```

Comment (rapzo) [reply package.json]: Ah! Should have payed more attention to how it was.
Nice one.

**Pull request merged**